src/transformer.js

```diff
@@ -30,12 +36,17 @@ const processAsync = async (source, filename, jestOptions) => {
 }
 
 /**
- * Starts a new process, so is higher overhead than processAsync.
+ * Starts a new process, so it has a higher overhead than processAsync.
  * However, Jest calls this method in CJS mode.
  */
 const processSync = (source, filename, jestOptions) => {
   const options = jestOptions && jestOptions.transformerConfig ? jestOptions.transformerConfig : {}
-  const { preprocess, rootMode, maxBuffer, showConsoleLog } = options
+  const { preprocess, rootMode, maxBuffer, showConsoleLog, debug } = options
+
+  if (debug) {
+    console.debug(`Running svelte-jester-transformer async in mode ${currentFileExtension}.`)
```

Comment:
should this be `sync` rather than `async`?
```suggestion
    console.debug(`Running svelte-jester-transformer sync in mode ${currentFileExtension}.`)
```